aiida_quantumespresso/utils/validation/trajectory.py

```diff
@@ -54,7 +53,7 @@ def verify_convergence_forces(trajectory, index=-1, threshold=None):
     try:
         forces = trajectory.get_array('forces')[index]
     except (KeyError, IndexError):
-        return None
```

Comment:
If I understand this right, you could previously specify `threshold_forces` even if no forces were created, but now it will raise. 

Is that correct? Or was this combination caught somewhere else also? Anyway, I think the change in behavior is fine if intentional. There's not too much sense in specifying a threshold for forces if those aren't computed.

Reply:
Yeah that's true, but I figure the old behavior did not make much sense and made it difficult to use. On top of that I introduced these functions in one of the alphas and only used them internally, so I feel its fine to change the behavior

Reply:
Yeah, I agree - the new behavior makes more sense.